tools/tune/src/src_generate.m

```diff
@@ -242,7 +242,7 @@
 end
```

Comment:
Looks OK but add a description of problem you fixed into commit message. Also your full name as  you prefer to write it to signed-off-by please. I got recently my signoff messed up similarly due to another Intel github project configured.

Reply:
OK, I will add the description.

Reply:
Thanks for adding it, but please follow git format, first line is a very short summary, then a space, then a full body description